actionview/test/template/render_test.rb

```diff
@@ -154,6 +154,13 @@ def test_render_raw_is_html_safe_and_does_not_escape_output
     assert_equal buffer, "<%= hello_world %>\n"
   end
 
+  def test_render_does_not_double_escape_strip_tags_output
+    helper = Class.new { include ActionView::Helpers::SanitizeHelper }.new
+    buffer = ActiveSupport::SafeBuffer.new
+    buffer << helper.strip_tags("<div>hello & goodbye</div>")
+    assert_equal("hello &amp; goodbye", buffer) # and not "hello &amp;amp; goodbye"
+  end
```

Comment:
Nit-pick: I'm not sure if this test belongs in this file.  Though no obvious alternatives jump out.  (`git grep SafeBuffer actionview/test` shows only this file and `actionview/test/template/javascript_helper_test.rb`.)  Perhaps we just rely on the on the added `actionview/test/template/sanitize_helper_test.rb` test, plus the [existing test in `activesupport/test/safe_buffer_test.rb`](https://github.com/rails/rails/blob/29298d2109e66b4c7ea04b50f04727161ec1687f/activesupport/test/safe_buffer_test.rb#L26-L29)?  (Though I don't feel strongly either way.)

Reply:
Ah, I missed the test that you linked (https://github.com/rails/rails/blob/29298d2109e66b4c7ea04b50f04727161ec1687f/activesupport/test/safe_buffer_test.rb#L26-L29). I'll update to remove this test (or actually move the parts I like into that test).